app/src/main.tsx

```diff
@@ -1,7 +1,6 @@
 import React from 'react'
 import ReactDOM from 'react-dom/client'
 import App from './App'
-import './index.css'
 
 ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
```

Comment:
Not sure if eslint is giving you any trouble with this but I believe this is the way React wants you to createRoot in version 18: https://github.com/BradyMitch/LaunchPad/blob/main/react-express-postgres/frontend/src/main.tsx

Reply:
I believe this was what Vite spawned on its own. I think I'll leave it as is for now, but if I end up needing to store the element, I'll convert it to that.